Apps/phdigitalguardianarc/digitalguardianarc_consts.py

```diff
@@ -0,0 +1,37 @@
+DG_CONFIG_SERVER_URL = 'server_url'
+DG_CONFIG_API_KEY = 'api_key'
+DG_ON_POLL = '/1.0/rest/1.0/export_profiles'
+DG_COMPONENT_LIST = '/1.0/remediation/lists'
+DG_WATCHLIST = '/rest/1.0/watchlists'
+DG_PARAM_LIST_FILES = 'get-files'
+DG_PARAM_IP = 'ip'
+DG_HEADER_URL = {'Content-Type': 'application/x-www-form-urlencoded', }
+DG_HEADER_JSON = {'Content-Type': 'application/json'}
+DG_CLIENT_HEADER = {'Authorization': '', 'Accept': 'application/json'}
+DG_PARAM_START_TIME = 'start_time'
+DG_PARAM_END_TIME = 'end_time'
+DG_QUERY_PARAM = 'srcip eq {srcip} or dstip eq {dstip}'
+DG_INVALID_START_TIME = "Parameter 'start_time' failed validation"
+DG_INVALID_END_TIME = "Parameter 'end_time' failed validation"
+DG_INVALID_TIME_RANGE = ("Invalid time range. 'end_time'"
```

Comment:
This is not used in code

Reply:
So are some of the other ones. Please remove if not used

Reply:
Removed the unused constants